ruller.go

```diff
@@ -298,6 +299,22 @@ func mergeMaps(rinfo *ruleInfo, sourceMap map[string]interface{}, destMapP *map[
 	}
 }
 
+func Middleware(next http.Handler) http.Handler {
```

Comment:
I suggest using cors middleware so that you can focus more on the parameters that may be passed to ruller for cors config. As it is meant to me production grade, the various parameters should be possible to be tweaked.

See an example os the lib usage at https://github.com/stutzlab/userme/blob/master/api.go, but I suggest you to pass the various parameters as program flags.

Reply:
Good example, unfortunately, it uses gin instead of mux, so I think we can't just use the code. But I got the main idea and I'll implement it.
Thank you so much